types/highcharts/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for Highcharts 5.0
+// Type definitions for Highcharts 7.1
```

Comment:
When upgrading the major version, please consider the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

Reply:
```suggestion
// Type definitions for Highcharts 5.0
```

Highcharts are supplying their own TypeScript definitions for Highcharts 7 and onwards. Also, this pull request does not include anything to lift the type definitions to version 7 either, so it is misleading. I suggest you leave this comment as it was.